gitlab-pipeline/stage/post.yml

```diff
@@ -36,7 +36,7 @@ mender-qa:failure:
     - cd ${CI_PROJECT_DIR}/${REPO_NAME}
   script:
     # Mark coverage report as done (mender-client).
-    - 'curl -k ${COVERALLS_WEBHOOK_URL}?repo_token=${COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
+    - 'curl -f ${COVERALLS_WEBHOOK_URL}?repo_token=${COVERALLS_TOKEN} -d "payload[build_num]=$(git rev-parse HEAD)&payload[status]=done"'
```

Comment:
Not important for now, but can we consider starting using long options for our code?

Reply:
I think in general I'm in favor, but not for `-f`. It's extremely common, and in fact we should use it on practically all `curl` calls we make. It's kind of like how `rm --recursive --force` is a bit much.

The preexisting one, `-k`, this could have been spelled out I think, as it's both less common, and more security sensitive.

I'll keep it in mind, I agree we should default to long options in most cases.

Reply:
I had to look it up, just saying :p

Reply:
A good time to remember it. 🙂 It should be used on every curl call, or else we won't catch errors.